hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -219,6 +219,8 @@
       swaccess: "ro",
       hwaccess: "hwo",
       hwext:    "true",
+      tags: [ // Life cycle internal HW will update status so could not auto-predict its value
+              "excl:CsrAllTests:CsrExclCheck"],
```

Comment:
can we test the reset value? If not, perhaps add comment to explain that too. Same for below 2 exclusions.

Reply:
we cannot because the reset value will set the idle bit to 1. But actually next two items can, I will update that. Thanks for the review

Reply:
do you mean field `READY`? there is no IDLE field

Reply:
This has not been updated yet. Michael will update that, in RTL it is called IDLE.

Reply:
I see. Thanks for the explanation.